homeassistant/components/ozw/climate.py

```diff
@@ -178,15 +182,7 @@ def hvac_mode(self):
     @property
     def hvac_modes(self):
         """Return the list of available hvac operation modes."""
-        if not self.values.mode:
-            return []
-        # Z-Wave uses one list for both modes and presets. Extract the unique modes
-        all_modes = []
-        for val in self.values.mode.value[VALUE_LIST]:
-            hass_mode = ZW_HVAC_MODE_MAPPINGS.get(val[VALUE_ID])
-            if hass_mode and hass_mode not in all_modes:
-                all_modes.append(hass_mode)
-        return all_modes
+        return list(self._hvac_modes.keys())
```

Comment:
Copying a dict to a list copies the keys by default. We can remove `.keys()`.

Reply:
fancy! fixed :-)